WORKSPACE

```diff
@@ -13,4 +13,4 @@ load("@io_bazel_rules_go//go:def.bzl", "go_rules_dependencies", "go_register_too
 load("@com_lyft_protoc_gen_validate//bazel:go_proto_library.bzl", "go_proto_repositories")
 go_proto_repositories(shared=0)
 go_rules_dependencies()
-go_register_toolchains()
+go_register_toolchains(go_version="host")
```

Comment:
This is going to do the wrong thing on most platforms, by using whatever random Go installation happens to exist in the environment instead of the hermetic versioned SDK downloaded by rules_go.

It looks like rules_go has supported detecting `linux_ppc64le` for several months (https://github.com/bazelbuild/rules_go/pull/1145), what does this particular change get us?

Reply:
yes, this is definitely not going to stay, sorry. it's just easier for me to test.

that change you posted to rules_go may not be in the rules_go version envoy pulls in, b/c if i don't change this for us, it pulls down the amd64 go binary.